src/core/ad.jl

```diff
@@ -433,3 +429,38 @@ MvNormal(m::TrackedVector{<:Real}, A::UniformScaling{<:Real}) = MvNormal(m, A.λ
 
 # zero mean,, constant variance
 MvNormal(d::Int, σ::TrackedReal{<:Real}) = MvNormal(zeros(d), σ)
+
+import Bijectors: link, invlink
+using Bijectors: Dirichlet, PDMatDistribution
+
+for F in (:link, :invlink)
+    @eval begin
+        $F(dist::Dirichlet, x::Tracker.TrackedArray, ::Type{Val{proj}}=Val{true}) where {proj} = Tracker.track($F, dist, x, Val{proj})
+        Tracker.@grad function $F(dist::Dirichlet, x::Tracker.TrackedArray, ::Type{Val{proj}}) where {proj}
+            x_data = Tracker.data(x)
+            y = $F(dist, x_data, Val{proj})
+            return  y,
+                    Δ -> begin
+                        out = ForwardDiff.jacobian(x -> $F(dist, x, Val{proj}), x_data)' * Δ
```

Comment:
Please sort out code formatting here. The following would be more inline with the style guide IMHO:
```julia
return y, Δ -> begin
    out = ForwardDiff.jacobian(x -> $F(dist, x, Val{proj}), x_data)' * Δ
    return (nothing, out, nothing)
end
```